gateway/src/apicast/policy/url_rewriting/apicast-policy.json

```diff
@@ -55,6 +55,61 @@
           },
           "required": ["op", "regex", "replace"]
         }
+      },
+      "query_args_commands": {
+        "description": "List of commands to apply to the query string args",
+        "type": "array",
+        "items": {
+          "type": "object",
+          "properties": {
+            "op": {
+              "description": "Operation to apply to the query argument",
+              "type": "string",
+              "oneOf": [
+                {
+                  "enum": ["add"],
+                  "title": "Set an argument if it does not exist"
```

Comment:
This is different from what we have in the headers policy. There it is "append".

Query string arguments are quite similar and we have to have some "append" operation.

Reply:
You're right. I forgot about values with several arguments.

I've adapted the code to take this case into account. The big changes are in `query_params.lua`. The other changes that I needed to make were basically adapting the rest of modules to use the correct names of the operations.